packages/database/lib/migrations/20241122212401_integration_backfill_missing_fields.cjs

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-var-requires */
+const path = require('node:path');
+const fs = require('node:fs');
+const yaml = require('js-yaml');
+
+exports.up = async function (knex) {
+    let providers;
+    const providersPath = path.join(__dirname, '..', '..', '..', 'shared', 'providers.yaml');
+    try {
+        providers = yaml.load(fs.readFileSync(providersPath, 'utf8'));
```

Comment:
I wish we could have a compile time `fileExist` check so we get an error here next time we move the providers file

Reply:
Yeah, I think it would actually make sense for us to make sure we're testing migrations from a clean state if possible to ensure that we catch that situation for self-installs.